packages/generate-bot/templates/test/Fixtures/config/valid-config.yml

```diff
@@ -0,0 +1 @@
+randomBoolean: true
```

Comment:
this folder should be lowercase, `fixtures`, rather than `Fixtures` (I noticed the generated tests fail due to the uppercase letter).